Formula/file-roller.rb

```diff
@@ -51,6 +69,6 @@ def post_install
   end
 
   test do
-    system "#{bin}/file-roller", "--version"
+    assert_match version.to_s, shell_output("#{bin}/file-roller --version 2>/dev/null")
```

Comment:
Since the version test passes with or without it, I figured I'd save people time and brain cycles re-reading that same gibberish every time they look at the CI, but there's no harm in removing the `2>/dev/null` if you'd like.